### ISSUE TITLE
Update from Feature File makes poor assumptions

### ISSUE THREAD

**Issue by adamlacoste** [hiptest/hiptest-issue-tracker#130]
When I use the Update from Feature File tool in HipTest to upload and parse a Gherkin feature file, I often find that the tool makes incorrect (and easily avoidable) assumptions about how to map each scenario step to a defined Action Word.

Suppose that I have the following Action Words defined in HipTest:
```
I click the "ELEMENT_NAME"
I click the "Nth" "ELEMENT_NAME"
```
The purpose of the first is to click some specific element I name in the scenario; the second is to click one particular element in a set of similar elements.  The quotes that HipTest Action Words use to delimit parameters make these two patterns distinct.

But when I upload a scenario that contains a step like this...
```
When I click the "log in button"
```
... what the Update from Feature File tool seems to do is strip the quotation marks and then try to parse the step without them.  What it usually ends up doing is selecting the second Action Word pattern listed above, with "log in" as the p1 value and "button" as the p2 value.  (Where it splits between the param values seems to be a bit arbitrary.)

This is ridiculous, because the gherkin feature file I uploaded made perfectly clear where the param values start and end by using the same quotation convention that HipTest uses.  If it would use those quotation marks to guide it, it would correctly identify this step as matching the first Action Word listed above, not the second.

**Comment by cbliard** [comment]
Hi Adam,

Indeed the action word matching does not yield its best results here. We'll get a closer look and come back with a fix.

Thank you for reporting this issue

**Comment by PhilipSwiatczak** [comment]
hi @adamlacoste, introducing text between arguments should help Hiptest uploader determine the boundary correctly. 
In my experience of using hiptest with java cucumber, listing two regular expression arguments one after another in step body without other characters in between causes cucumber itself to get confused about the boundary.
Consider your example from StepDefinition regex level:
 ("^I click the \"(.*)\" \"(.*)\"$")
I may be wrong, but I don't think Hiptest allows specifying how to exactly formulate regex expression in its free text argument fields. 
Therefore code interpreter reads (start with "I click the" followed by "any text of any length" followed by "any text of any length".

**Comment by adamlacoste** [comment]
Certainly, those regex patterns would be ambiguous; that's why we structure ours like this:

```
/I click the "([^"]*)"$/i
/I click the "([0-9]*)(?:st|nd|rd|th)" "([^"]*)"$/i
```

This leaves no ambiguity between the step definitions.

**Comment by cbliard** [comment]
Hi Adam,

The issue has finally been solved. Sorry for the long delay.
Can you confirm that it now works as expected for all your patterns?

Regards

**Comment by cbliard** [comment]
The [import feature file](https://hiptest.com/docs/import-feature-file/) functionality is available only when these two conditions are met:
  - You have subscribed to a Pro plan (during the early days of this feature, it was available for all plans)
  - You have activated "Automation Ready" mode in project settings

Then you should be able to import a feature file from within a folder.

To update a folder from a feature file:
![image](https://user-images.githubusercontent.com/176055/48250854-07fdb300-e400-11e8-83f6-0eea4310eb8e.png)

To create a new subfolder from a feature file:
![image](https://user-images.githubusercontent.com/176055/48250900-24015480-e400-11e8-9ac4-f470b03a0e09.png)
